spec/yaml/swiftnav/sbp/system.yaml

```diff
@@ -215,6 +214,59 @@ definitions:
             fill: SubSystemReport
             desc: Reported status of individual subsystems
 
+ - StatusJournalItem:
+    short_desc: Subsystem Status report
+    embedded_type: true
+    desc: >
+      Reports the uptime and the state of a subsystem via generic and specific
+      status codes.  If the generic state is reported as initializing, the
+      specific state should be ignored.
+    fields:
+        - uptime:
+            type: u32
+            desc: Milliseconds since system startup
+        - report:
+            type: SubSystemReport
```

Comment:
Embedded the SubSystemReport here since the old definition was a copy but added the uptime field.